app/src/main/java/org/mozilla/fenix/library/downloads/DownloadFragmentStore.kt

```diff
@@ -46,7 +53,9 @@ sealed class DownloadFragmentAction : Action {
  */
 data class DownloadFragmentState(
     val items: List<DownloadItem>,
-    val mode: Mode
+    val mode: Mode,
+    val pendingDeletionIds: Set<String>,
+    val isDeletingItems: Boolean
```

Comment:
Could we use `pendingDeletionIds` to determine `isDeletingItems`? For example: `isDeletingItems = pendingDeletionIds.isEmpty`?